app/services/claim_fast_tracking/max_rating_annotator.rb

```diff
@@ -49,6 +37,10 @@ def self.get_ratings(diagnostic_codes)
       nil
     end
 
-    private_class_method :annotate_disabilities_api_disabled, :annotate_disabilities_api_enabled, :get_ratings
+    def self.eligible_for_request?(dc)
+      Flipper.enabled?(:disability_526_maximum_rating_api_all_conditions) || SELECT_DISABILITIES.include?(dc)
```

Comment:
The MaxRatingAnnotator is user-agnostic, so it makes sense to check for a global Flipper enable, but it does mean that if we ever want to roll this out to only a subset of users, we'd have to revisit this code. Just something to keep in mind, @amylai-nava.